strings.txt

```diff
@@ -16244,3 +16246,19 @@
     tags = ios, android
     en = Unknown Place
     ru = Неизвестное место
+
+  [editor_other_info]
+    tags = ios, android
+    en = Other information about place
+
+  [editor_detailed_description_hint]
+    tags = ios, android
+    en = Detailed description
+
+  [editor_detailed_description]
+    tags = ios, android
+    en = Suggested changes will be send to OpenStreetMap...
```

Comment:
Заглушка*
> On Apr 21, 2016, at 4:34 PM, Dmitry Yunitsky <dmitry.yunitsky@gmail.com> wrote:
> 
> Это временная заглушила для длинного текста, перевода которого нет.
>> On Apr 21, 2016, at 4:02 PM, Alex Zolotarev <notifications@github.com <mailto:notifications@github.com>> wrote:
>> 
>> In strings.txt <https://github.com/mapsme/omim/pull/2947#discussion_r60575032>:
>> 
>> > @@ -16244,3 +16246,19 @@
>> >      tags = ios, android
>> >      en = Unknown Place
>> >      ru = Неизвестное место
>> > +
>> > +  [editor_other_info]
>> > +    tags = ios, android
>> > +    en = Other information about place
>> > +
>> > +  [editor_detailed_description_hint]
>> > +    tags = ios, android
>> > +    en = Detailed description
>> > +
>> > +  [editor_detailed_description]
>> > +    tags = ios, android
>> > +    en = Suggested changes will be send to OpenStreetMap...
>> Многоточие должно быть одним юникодным символом.
>> 
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2947/files/44b3f477a158544a38212dbfd95100ddbb3c305c#r60575032>
>